playbook/app/pb_kits/playbook/playbook-doc.js

```diff
@@ -94,9 +94,10 @@ import * as Toggle from 'pb_toggle/docs'
 import * as Typeahead from 'pb_typeahead/docs'
 import * as User from 'pb_user/docs'
 import * as UserBadge from 'pb_user_badge/docs'
+import * as Walkthrough from 'pb_walkthrough/docs'
 import * as WeekdayStacked from 'pb_weekday_stacked/docs'
 
-WebpackerReact.registerComponents({
+WebpackerReact.setup({
```

Comment:
Wait a sec. Why did this line change? We should revert this.

Reply:
@thestephenmarshall I believe this was pulled in because of carlos's PR at the time

Reply:
it should still be reverted... it should match whatever is on master, @kre8sions ...

You can run `git checkout master -- playbook/app/pb_kits/playbook/playbook-doc.js` and then add, commit, and push. And you will no longer see the file changed.

Reply:
^^ it might be `git checkout origin master -- playbook/app/pb_kits/playbook/playbook-doc.js` just fyi.

Reply:
If you resolve your merge conflicts too, it might fix these.

Reply:
+1 Yeh probably just a borked resolution

Reply:
@thestephenmarshall ujs is not needed anymore that is why setup is the new method